resources/shaders/rt_hybrid.hlsl

```diff
@@ -95,24 +95,25 @@ float3 TraceReflectionRay(float3 origin, float3 norm, float3 direction, uint ran
 		return skybox.SampleLevel(s0, SampleSphericalMap(direction), 0).rgb;
 	}
 
-	origin += norm * EPSILON;
+	//origin += norm * EPSILON;
+	float3 colorz = float3(0, 0, 0);
```

Comment:
wtf really

Reply:
This was unintended and managed to slip through my commits.